app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.kotlinplayground
+
+import kotlin.random.Random
+
+class StudentNameSantorini() {
+    val numberOfStudentsInGroup: Int = 100
```

Comment:
This values should be private and in the class constructor

Reply:
@tagirf  please take a look one more time 
fixed